docs/src/main/diataxis/_examples/acme-serve-http-requests-tutorial.adoctxt

```diff
@@ -0,0 +1,125 @@
+////
```

Comment:
Why the weird extension for this adoc?

Reply:
So that it isn't formatted when it is included at the end of the associated tutorial.